assignments/objects.js

```diff
@@ -33,13 +66,30 @@ const example = {
 
 // Antonietta's Gender
 
+console.log(intern1.name)
+console.log(intern2.id)
+console.log(intern3.email)
+console.log(intern4.name)
+console.log(intern5.gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+intern2.spak = function() {
+  return 'Hello, my name is Kennan!';
```

Comment:
Needs to be done automatically! Manually writing Kennan in the string is pointless if the intern changes. Use interpolation `${here}`.